src/grelu/interpret/score.py

```diff
@@ -205,6 +205,98 @@ def get_attributions(
     return attributions  # N, 4, L
 
 
+def _add_tomtom_to_modisco_report(
+    modisco_dir: str,
+    tomtom_results: pd.DataFrame,
+    meme_file: str,
+    top_n_matches: int,
+) -> None:
+    """
+    Modified from https://github.com/jmschrei/tfmodisco-lite/blob/main/modiscolite/report.py#L245
```

Comment:
can you add the permalink: https://github.com/jmschrei/tfmodisco-lite/blob/3c6e38f/modiscolite/report.py#L245

Reply:
also, would it make more sense to put this function and run_modisco in a separate modisco.py file? Does not fit into score.py thematically I feel.

Reply:
done